pkg/cloud/aws/services/cloudformation/bootstrap.go

```diff
@@ -176,6 +176,7 @@ func controllersPolicy(accountID string) *iam.PolicyDocument {
 					"ec2:DescribeVpcs",
 					"ec2:DetachInternetGateway",
 					"ec2:DisassociateRouteTable",
+					"ec2:ModifyInstanceAttribute",
```

Comment:
@rudoi can you please clarify what operation needs to perform this action?

Reply:
Sure thing. It is called when updating instance security groups:

https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/870d03d2d5a25a407b335d3f0b702227dfd7cd59/pkg/cloud/aws/services/ec2/instances.go#L414

Reply:
I also added DeleteTags, which is called here:

https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/870d03d2d5a25a407b335d3f0b702227dfd7cd59/pkg/cloud/aws/services/ec2/instances.go#L461

Reply:
Thanks for clarifying!